docs/configuration/data-validation.md

```diff
@@ -108,3 +109,28 @@ PROTECTION_RULES = {
     ]
 }
 ```
+
+### Configuration via Web UI
+Example prevents the deletion of a device entry:
+
+```python
+{
+    "dcim.device": [
+        {
+            "status": {
+                "eq": "decommissioning"
+            }
+        }
+    ]
+}
+```
+[![NetBox UI](./media/screenshots/Configuration-History-Protection-rules-Example.png)](./media/screenshots/Configuration-History-Protection-rules-Example.png)
```

Comment:
Both of your images are broken. Please be sure to render the documentation locally with `mkdocs serve` and verify your changes before submitting a PR.